src/index.ts

```diff
@@ -185,6 +193,23 @@ async function activate(
     gitPlugin.title.icon = gitIcon;
     gitPlugin.title.caption = 'Git';
 
+    // Add the commands to the command palette
+    const category = 'Git Operations';
+    [
+      CommandIDs.gitToggleSimpleStaging,
+      CommandIDs.gitToggleDoubleClickDiff,
+      CommandIDs.gitOpenGitignore,
+      CommandIDs.gitShowDiff,
+      CommandIDs.gitInit,
+      CommandIDs.gitClone,
```

Comment:
You must not add this one here as it is defined in another plugin.

```suggestion
```

Instead you should add the same logic in the clone plugin there:

https://github.com/jupyterlab/jupyterlab-git/blob/dda8e82b9b2751e1b2006f71cb101accef261d10/src/cloneCommand.ts#L20

Reply:
> You must not add this one here as it is defined in another plugin.
> 
> Instead you should add the same logic in the clone plugin there:
> 
> https://github.com/jupyterlab/jupyterlab-git/blob/dda8e82b9b2751e1b2006f71cb101accef261d10/src/cloneCommand.ts#L20

@fcollonval Hi, thanks for the feedback. Could you kindly elaborate a bit for me on what you meant by "Instead you should add the same logic in the clone plugin there". Thank you once again.

Reply:
`gitCloneCommandPlugin` is an extension plugin as the `plugin` you modified there:

https://github.com/jupyterlab/jupyterlab-git/blob/fba3e2fe1def7d9f051221cf2d7e2aac26652be5/src/index.ts#L44

So you should also add the `ICommandPalette` token as requirement and then call,

```ts
palette.addItem({ command: CommandIDs.gitClone, category: trans.__('Git Operations') })
```